javascript/processingFunctions/processSnippetPdf.js

```diff
@@ -124,6 +124,10 @@ export const processSnippetPdf = (node, writeTo) => {
   const jsLonelySnippet = node.getElementsByTagName("JAVASCRIPT_LONELY")[0];
   const jsSnippet = node.getElementsByTagName("JAVASCRIPT")[0];
   const jsOutputSnippet = node.getElementsByTagName("JAVASCRIPT_OUTPUT")[0];
+  const allowBreakAfter =
```

Comment:
name of the constant "allowBreakAfter" is confusing. This should be "forceBreakAfter" isn't it?